scoring/src/main/scala/ai/h2o/sparkling/ml/params/H2OWord2VecExtraParams.scala

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ai.h2o.sparkling.ml.params
+
+import ai.h2o.sparkling.ml.models.H2OFeatureEstimatorBase
+import org.apache.spark.sql.types.{ArrayType, FloatType, StructField, StructType}
+
+trait H2OWord2VecExtraParams extends H2OFeatureEstimatorBase with HasOutputCol with HasInputColOnMOJO {
+
+  override protected def validate(schema: StructType): Unit = {
```

Comment:
Don't we need to add additional type checks for inputCol? E.g array column?

Reply:
I added the "array of strings" check, wdyt?

Reply:
nice catch!